crates/wasmtime/src/trampoline.rs

```diff
@@ -77,6 +77,7 @@ fn create_handle(
                     as *const VMExternRefActivationsTable
                     as *mut _,
                 stack_map_registry: store.stack_map_registry() as *const StackMapRegistry as *mut _,
+                limiter: None,
```

Comment:
I think this needs to be specified? Otherwise are memories created via `Memory::new` (or tables) limited via the limiter?

Reply:
Good point.  I'll fix.